### PR TITLE
Update rise-storage version

### PR DESCRIPTION
**Pull request by donnapep**
* Clean up gulpfile.js.
* Clean up package.json.

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees Please review. Thx.

File: gulpfile.js [hunk 1]
```diff
@@ -28,13 +27,12 @@
...
-  gulp.task("clean", function () {
-    return gulp.src("dist", {read: false})
-      .pipe(rimraf());
+  gulp.task("clean", function (cb) {
+    del(["./dist/**"], cb);
   });
 
   gulp.task("config", function() {
-    var env = process.env.NODE_ENV || "dev";
+    var env = process.env.NODE_ENV || "prod";
```
Comment (stulees) [inline gulpfile.js]: @donnapep I know we need to ensure we build and commit a production version to `dist/`, but was this intentional?

Comment (donnapep) [inline gulpfile.js]: @stulees Yes, I want to ensure that I don't accidentally commit a development version. Do you know of a better way?

Comment (stulees) [inline gulpfile.js]: @donnapep I think this is probably best.

**Comment by stulees** [comment]
@donnapep :+1:

**Comment by donnapep** [comment]
@stulees Check it out. Notice the placement of `sinon.useFakeTimers()` in `widget-url.js`. It needs to fire after Casper has opened the page but before any timers are set. I could probably go and do a similar thing in the Storage version, but I'm going to leave it as is and only change it if we encounter problems.

Please review. Thx.

**Comment by stulees** [comment]
@donnapep Nice one on getting fakeTimers to work. :+1:

**Comment by donnapep** [comment]
@stulees Please review last commit. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Comment by donnapep** [comment]
* Changed refresh interval to 5 minutes.
* Set URL component `type` to `single-file`.
* Update Storage modal URLs.
* Update Polymer, webcomponents and rise-storage.

@stulees Please review. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Pull request merged**